forge-gui/res/cardsfolder/k/karma.txt

```diff
@@ -4,5 +4,7 @@ Types:Enchantment
 T:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ Player | TriggerZones$ Battlefield | Execute$ TrigDamage | TriggerDescription$ At the beginning of each player's upkeep, CARDNAME deals damage to that player equal to the number of Swamps they control.
 SVar:TrigDamage:DB$ DealDamage | Defined$ TriggeredPlayer | NumDmg$ X
 SVar:X:Count$Valid Swamp.ActivePlayerCtrl
-AI:RemoveDeck:Random
```

Comment:
the AI tag should be kept, when generating random deck AI can't know what cards opponent will have
same for below